src/wp-includes/class-wp-block-type.php

```diff
@@ -117,6 +117,14 @@ class WP_Block_Type {
 	 */
 	public $variations = array();
 
+	/**
+	 * Custom CSS selectors for theme.json style generation.
+	 *
+	 * @since 6.2.0
```

Comment:
It's probably too late so:
```suggestion
	 * @since 6.3.0
```

Reply:
Thanks, agreed. It definitely missed 6.2 🙂